nano/core_test/unchecked_map.cpp

```diff
@@ -261,8 +261,8 @@ TEST (unchecked, multiple_get)
 		unchecked2.push_back (i.block->hash ());
 	}
 	// Asserts the payloads where correctly saved
-	ASSERT_TRUE (std::find (unchecked2.begin (), unchecked2.end (), block1->hash ()) != unchecked2.end ());
```

Comment:
Leave it this as to was, iterators and equality asserts could cause problems. Leave as it was .